XamlControlsGallery/ControlPages/CreateMultipleWindowsPage.cs

```diff
@@ -0,0 +1,62 @@
+﻿//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+using AppUIBasics.Data;
+using AppUIBasics.Helper;
+using System.Collections.Generic;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace AppUIBasics.ControlPages
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class CreateMultipleWindowsPage : Page
+    {
+        private IEnumerable<ControlInfoDataGroup> _groups;
+
+        public CreateMultipleWindowsPage()
+        {
+            this.InitializeComponent();
+        }
+        public IEnumerable<ControlInfoDataGroup> Groups
```

Comment:
Consider adding a comment explaining what this is for and who is consuming it.

Reply:
Removed